README.md

```diff
@@ -0,0 +1,11 @@
+ament_lint contains a number of packages that perform linting tasks. Those linters can be used both from the command line and from CMake.
```

Comment:
FYI we usually try to only have one sentence per line in documentation-related documents. This minimizes the git diff when we make changes, which makes maintenance easier.

So for this line (and some of the lines below):

```suggestion
ament_lint contains a number of packages that perform linting tasks.
Those linters can be used both from the command line and from CMake.
```